modules/apps/asset/asset-list-test/src/testIntegration/java/com/liferay/asset/list/asset/entry/provider/test/AssetListAssetEntryProviderTest.java

```diff
@@ -73,6 +107,69 @@ public void setUp() throws Exception {
 			_group.getGroupId(), TestPropsValues.getUserId());
 	}
 
+	@Test
+	public void testCombineSegmentsOfDynamicCollection() throws Exception {
+		Dictionary<String, Object> dictionary =
+			HashMapDictionaryBuilder.<String, Object>put(
+				"enabled", true
+			).build();
+
+		try (ConfigurationTemporarySwapper configurationTemporarySwapper =
+				new ConfigurationTemporarySwapper(
+					"com.liferay.asset.list.web.internal.configuration." +
+						"FFCollectionsVariationsPrioritizationConfiguration",
+					dictionary)) {
```

Comment:
As dictionary is only used here, maybe you can use the HashMapDictionaryBuilder directly

Reply:
Done